tests/integration/generated/test_complete_run_chrysalis.cfg

```diff
@@ -43,10 +43,12 @@ years = "1850:1854:2",
   vars = "PRECT"
 
   [[ atm_monthly_glb ]]
+  active = True
   frequency = "monthly"
   input_files = "eam.h0"
   input_subdir = "archive/atm/hist"
   mapping_file = "glb"
+  vars = "FSNTOA,FLUT,FSNT,FLNT,FSNS,FLNS,SHFLX,QFLX,TAUX,TAUY,PRECC,PRECL,PRECSC,PRECSL,TS,TREFHT,CLDTOT,CLDHGH,CLDMED,CLDLOW,U,CO2,CO2_FFF,CO2_OCN,CO2_LND,TMCO2,TMCO2_FFF,TMCO2_LND,TMCO2_OCN,TOTSOMC,TOTECOSYSC,NEE,TOTVEGC"
```

Comment:
The extra variables cause the following errors for the `atm_monthly_glb` subtask of `ts`:
```
ncrcat: ERROR nco_xtr_mk() reports user-supplied variable name or regular expression 'TMCO2_LND' is not in and/or does not match contents of input file
ncrcat: ERROR nco_xtr_mk() reports user-supplied variable name or regular expression 'TOTSOMC' is not in and/or does not match contents of input file
ncrcat: ERROR nco_xtr_mk() reports user-supplied variable name or regular expression 'CO2' is not in and/or does not match contents of input file
ncrcat: ERROR nco_xtr_mk() reports user-supplied variable name or regular expression 'CO2_OCN' is not in and/or does not match contents of input file
ncrcat: ERROR nco_xtr_mk() reports user-supplied variable name or regular expression 'TMCO2' is not in and/or does not match contents of input file
ncrcat: ERROR nco_xtr_mk() reports user-supplied variable name or regular expression 'TOTECOSYSC' is not in and/or does not match contents of input file
ncrcat: ERROR nco_xtr_mk() reports user-supplied variable name or regular expression 'TMCO2_FFF' is not in and/or does not match contents of input file
ncrcat: ERROR nco_xtr_mk() reports user-supplied variable name or regular expression 'TMCO2_OCN' is not in and/or does not match contents of input file
ncrcat: ERROR nco_xtr_mk() reports user-supplied variable name or regular expression 'CO2_LND' is not in and/or does not match contents of input file
ncrcat: ERROR nco_xtr_mk() reports user-supplied variable name or regular expression 'TOTVEGC' is not in and/or does not match contents of input file
ncrcat: ERROR nco_xtr_mk() reports user-supplied variable name or regular expression 'CO2_FFF' is not in and/or does not match contents of input file
ncrcat: ERROR nco_xtr_mk() reports user-supplied variable name or regular expression 'NEE' is not in and/or does not match contents of input file
```

Reply:
These variables I believe are specialized BGC/HES output, the error is expected if the variables are not included in the test datasets.

Reply:
Maybe @BunnyVon could provide some datasets for testing here.